patches/@endo+ses-ava+0.2.27.patch

```diff
@@ -0,0 +1,76 @@
+diff --git a/node_modules/@endo/ses-ava/src/ses-ava-test.js b/node_modules/@endo/ses-ava/src/ses-ava-test.js
+index 36038ad..ff648d7 100644
+--- a/node_modules/@endo/ses-ava/src/ses-ava-test.js
++++ b/node_modules/@endo/ses-ava/src/ses-ava-test.js
+@@ -57,6 +57,7 @@ const testerMethodsWhitelist = [
+   'serial',
+   'only',
+   'skip',
++  'todo',
+ ];
+ 
+ /**
+@@ -67,15 +68,18 @@ const testerMethodsWhitelist = [
+ const wrapTester = (testerFunc, logger = defaultLogger) => {
+   /** @type {TesterFunc} */
+   const testerWrapper = (title, implFunc, ...otherArgs) => {
+-    /** @type {ImplFunc} */
+-    const testFuncWrapper = t => {
+-      harden(t);
+-      return logErrorFirst(implFunc, [t, ...otherArgs], 'ava test', logger);
+-    };
+-    if (implFunc && implFunc.title) {
+-      testFuncWrapper.title = implFunc.title;
++    if (implFunc) {
++      const testFuncWrapper = t => {
++        harden(t);
++        return logErrorFirst(implFunc, [t, ...otherArgs], 'ava test', logger);
++      };
++      if (implFunc.title) {
++        testFuncWrapper.title = implFunc.title;
++      }
++      otherArgs.unshift(testFuncWrapper);
+     }
+-    return testerFunc(title, testFuncWrapper, ...otherArgs);
++    /** @type {ImplFunc} */
++    return testerFunc(title, ...otherArgs);
+   };
+   return testerWrapper;
+ };
+@@ -105,9 +109,9 @@ const wrapTester = (testerFunc, logger = defaultLogger) => {
+  * that eventually rejects, the error is first sent to the `console` before
+  * propagating into `rawTest`.
+  *
+- * @param {TesterInterface} avaTest
++ * @param {import('ava').TestInterface} avaTest
```

Comment:
Are we confident that our emulation satisfies the entire ava interface now? Or, is this a convenient fiction that improves the debug experience?

Reply:
You're right we don't. That's why this still returns `TestInterface`. But the `avaTest` param is indeed from Ava.

Marking this as Draft because my refactor broke some case.

Reply:
Confident now that it satisfies because it passes through whatever it doesn't override. I typed the param as generic so that it wraps whatever version of Ava was passed in.